packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -439,17 +443,45 @@ export default function DataviewsPatterns() {
 		return filterSortAndPaginate( patterns, viewWithoutFilters, fields );
 	}, [ patterns, view, fields, type ] );
 
-	const actions = useMemo(
-		() => [
+	const history = useHistory();
+	const onActionPerformed = useCallback(
+		( actionId, items ) => {
+			if ( actionId === 'edit-post' ) {
+				const post = items[ 0 ];
+				history.push( {
+					postId: post.id,
+					postType: post.type,
+					categoryId,
+					categoryType: type,
+					canvas: 'edit',
+				} );
+			}
+		},
+		[ history ]
```

Comment:
This is missing deps 'categoryId' and 'type', or was it intentional?

Reply:
Oh, thanks for raising, fix at https://github.com/WordPress/gutenberg/pull/60784